style.css

```diff
@@ -0,0 +1,225 @@
+/*==========================================================
+    VARIABLES
+==========================================================*/
+:root {
+    --news-red: #f72210;
+    --ethic-white: #fff;
+}
+/*==========================================================
+    GENERAL TAGS
+==========================================================*/
+body {
+    font-family: Arial,Helvetica,sans-serif;
+}
+
+.hid {
+    display: none;
+}
+
+body.container {
+    max-width: 1290px;
+    overflow-x: hidden;
+}
+
+h1, h2, h3, h4, h5 {
+    font: bold 19px/19px "Roboto Condensed",sans-serif;
+}
+
+p {
+    margin: 0;
+}
+
+ul, ol {
+    margin: 0;
+    list-style: none;
+}
+
+.fs-1-3 {
+    font-size: 1.3rem;
+}
+
+.txt-red {
+    color: var(--news-red)
+}
+
+.max-1290 {
+    max-width: 1290px;
+}
+
+section.max-1290 {
+    max-width: 1600px !important;
+}
+
+.bg-secondary {
+    background-color: #f4f5f6 !important;
+}
+
+[src="./assets/slides.png"] {
+    left: 0;
+    bottom: 0;
+}
+
+.slide-num {
+    bottom: 3%;
+    left: 12%;
+}
+
+.img-tag {
+    left: 0;
+    bottom: 0;
+}
+
+.image-container {
+    overflow: auto;
+}
+
+.image-container-100 {
+    height: 100%;
+}
+
+.slides {
+    bottom: 0;
+    left: 0;
+    width: 100%;
+}
+
+.min-301 { 
+    min-height: 301px;
+}
+
+/*==========================================================
+    CUSTOM ICONS
+==========================================================*/
+.icon-n {
+    height: 28px;
+    width: 29px;
+    margin-right: 10px;
+    display: block;
+    background: url("https://g.newsweek.com/www/images/logo-n1.svg") center/contain no-repeat var(--news-red);
+}
+
+.icon-slide {
+    box-shadow: 2px 2px 0 0 var(--ethic-white), 5px 5px 0 0 var(--news-red);
+}
+
+
+/*==========================================================
+    HEADER & NAV
+==========================================================*/
+.nav {
+    background-color: var(--news-red);
+}
+
+.logo {
+    max-width: min-content;
+}
+
+.nav-logo {
+    height: 60px;
+    width: 396px;
+}
+
+.nav-ul {
+    background-color: var(--ethic-white);
+}
+
+.ul-nav {
+    max-width: max-content;
+}
+
+.search-input {
+    width: 60px;
+    font-size: 0.9rem;
+}
+
+.nav-white {
+    overflow: hidden;
+    clip-path: polygon(0% 0%, 90% 0%, 92% 30%, 100% 30%, 100% 100%, 0% 100%)
+}
+
+/*==========================================================
+    FOOTER
+==========================================================*/
+.footer-logo {
+    width: 236px;
+    height: 36px;
+}
+
+.slanted-img-top {
+    top: -20px;
+    left: -20px;
+}
+
+.slanted-img-bottom {
+    left: 50px;
+}
+
+/*==========================================================
+    MEDIA QUERIES
+==========================================================*/
+@media only screen and (min-width: 760px) {
+    main:first-of-type {
+        height: 2150px;
+    }
+    
+    main #main-section {
+        height: 2020px !important;
```

Comment:
Does deleting this `!important` affects this element? :thinking:

Reply:
Nope it does not, deleted.
>